Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -2271,6 +2271,15 @@ void NetworkProcess::terminateRemoteWorkerContextConnectionWhenPossible(RemoteWo
     }
 }
 
+void NetworkProcess::runningOrTerminatingServiceWorkerCountForTesting(PAL::SessionID sessionID, CompletionHandler<void(unsigned)>&& completionHandler)
+{
+    auto* session = networkSession(sessionID);
+    if (!session)
+        return completionHandler(0);
+
+    completionHandler(session->ensureProtectedSWServer()->runningOrTerminatingCount());
```

Comment:
I would tend to use ensureSWServer() or swServer(), hopefully this should be aligned with our C++ guidelines.

Reply:
Let me see what the Safer C++ checker says. I'm actually not sure why the first call in the chain is allowed since `session` is a raw pointer and `ensureProtectedSWServer()` (and `ensureSWServer()`) seem like non-trivial member functions.